data/schemas/keyboard.jsonschema

```diff
@@ -372,6 +372,12 @@
                 }
             }
         },
+        "key_cancellation": {
```

Comment:
Might also be worth removing this schema change (and coresponding mapping changes too), as it currently doesnt offer anything more than `features.key_cancellation = true`.